backend/portal/api/curation/v1/curation/collections/collection_id/datasets/dataset_id/actions.py

```diff
@@ -37,7 +37,10 @@
 def get(collection_id: str, dataset_id: str = None):
     business_logic = get_business_logic()
 
+    # Double lookup
```

Comment:
nit - this comment would be more beneficial if it explained why we need the double lookup, as opposed to just saying that we are implementing it. That way in the future someone will know not to refactor / why it was put here in the first place